src/metrics/wcc.rs

```diff
@@ -40,16 +39,18 @@ pub(crate) fn wcc_plain_function(
         .try_fold(0., |acc, (i, line)| -> Result<f64> {
             // Check if the line is null
             let is_null = if is_covdir {
-                line.as_i64().ok_or(Error::ConversionError())? == -1
+                // This unwrap can't panic because if `is_covdir` is true then `line` is certainly a `Some(i32)`
```

Comment:
Instead of an `unwrap`, can we use an `if let` with a boolean value inside the branch that returns `true` or `false`?